src/assets/data/mock-compliance.json

```diff
@@ -0,0 +1,311 @@
+{ "dataset": [
```

Comment:
Use the current data that is the API for the "ACCELEROMETER" as an example to form this mock data.

```json
{
  "header": {
    "subjectId": "MRC01",
    "sourceId": "00:07:80:1F:17:6B",
    "source": "EMPATICA",
    "sensor": "ACCELEROMETER",
    "descriptiveStatistic": "AVERAGE",
    "unit": "G",
    "timeFrame": "TEN_SECOND",
    "effectiveTimeFrame": {
      "startDateTime": "2017-06-16T12:12:30Z",
      "endDateTime": "2017-06-16T15:11:20Z"
    }
  },
  "dataset": [
    {
      "sample": {
        "x": -0.35321180555555554,
        "y": -0.02673611111111111,
        "z": 0.9500868055555556
      },
      "startDateTime": "2017-06-16T12:12:30Z"
    },
    {
      "sample": {
        "x": -0.4702240566037736,
        "y": -0.3223270440251572,
        "z": 0.4982802672955975
      },
      "startDateTime": "2017-06-16T12:12:40Z"
    },
    {
      "sample": {
        "x": -0.5686910377358491,
        "y": -0.5421088836477987,
        "z": 0.06436713836477988
      },
      "startDateTime": "2017-06-16T12:12:50Z"
    },
    {
      "sample": {
        "x": -0.7989004629629629,
        "y": 0.04933449074074074,
        "z": -0.06978202160493827
      },
      "startDateTime": "2017-06-16T12:13:00Z"
    }
  ]
}
```